low-level/files/etc/rsyslog.d/1-ssplsbfwd.conf

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+# Redirecting sspl support bundle logs to specific file for having concentrated debugging logs.
+# Rsyslog TCP and UDP server.
+
+# Provisioned node itself UDP & TCP receptions are enabled in /etc/rsyslog.conf
+# Having TCP & UDP reception here for SSPL will make conflicts with rsyslog.
+# So commented below the TCP and UDP server.
+
+# Star Rsyslog server on port 514
+$ModLoad imtcp
+$InputTCPServerRun 514
+
+$ModLoad imudp
+$UDPServerRun 514
```

Comment:
`# Provisioned node itself UDP & TCP receptions are enabled in /etc/rsyslog.conf` As mentioned.
Enable these TCP, UDP configs only if required.

Reply:
Referred to rsyslog sspl.conf and kept the configuration same.
could you explain what conf changes required exactly?

Reply:
Just check /var/log/message is not receiving any message on support bundle generation. 
If it happens comment below lines.
```
$ModLoad imtcp
$InputTCPServerRun 514

$ModLoad imudp
$UDPServerRun 514
```
Provisioner used to enable these in rsyslog.conf.